src/zajecia20/RegistryForm.html

```diff
@@ -0,0 +1,99 @@
+                <input type="text" class="form-control" id="inputSurname" placeholder="Wpisz nazwisko">
+            </div>
+
+            <div class="form-group row">
+                <div class="col">
+                    <label class="row">Płeć</label>
+                </div>
+                <div class="col">
+                    <div class="custom-control custom-radio">
+                        <input type="radio" class="custom-control-input" id="defaultChecked"
+                               name="exampleRadio1" checked>
+                        <label class="custom-control-label" for="defaultChecked">Kobieta</label>
+                    </div>
+
+                    <div class="custom-control custom-radio">
+                        <input type="radio" class="custom-control-input" id="defaultUnchecked" name="exampleRadio2">
+                        <label class="custom-control-label" for="defaultUnchecked">Mężczyzna</label>
+                    </div>
+                </div>
+                <div class="col-8"></div>
+            </div>
+
+            <div class="form-group row">
+                <label for="emailInput">Adres email</label>
+                <input type="email" class="form-control" id="emailInput" placeholder="Wpisz adres email">
+            </div>
+
+            <div class="form-group row">
+                <label for="passwordInput">Hasło</label>
+                <input type="password" class="form-control" id="passwordInput" placeholder="Hasło">
+                <small id="passHelp" class="form-text text-muted">Min. 8 znaków.</small>
+            </div>
+
+            <div class="form-group row">
+        <span class="form-group align-top mr-3">
+            <label for="inputCity">Miasto</label>
+            <input type="text" class="form-control" id="inputCity">
+        </span>
+                <span class="form-group align-top mr-3">
+            <label>Województwo</label>
+
+        <select class="custom-select" required="required">
+            <option selected="selected" disabled="disabled">Wybierz...</option>
+            <option>dolnośląskie</option>
+            <option>mazowieckie</option>
+            <option>lubuskie</option>
+        </select>
+        </span>
+
+                <span class="form-group align-top">
+        <label for="inputPostCode">Kod pocztowy</label>
+        <input type="text" class="form-control" id="inputPostCode">
+    </span>
+            </div>
+            <div class="custom-control custom-checkbox row">
+                <input type="checkbox" class="custom-control-input" id="acceptCheckbox">
+                <label class="custom-control-label" for="acceptCheckbox">
+                    Akceptuje
+                    <a href="#" class=" stretched-link">regulamin</a>
+                </label>
+            </div>
+
+            <div class="custom-control custom-checkbox row">
+                <input type="checkbox" class="custom-control-input" id="marketingCheckbox">
```

Comment:
A to miało być od razu zaznaczone - po prostu wystarczyło dodać `checked`.

Reply:
poprawione